org.dupot.littleadventure.yml

```
@@ -0,0 +1,28 @@
+id: org.dupot.littleadventure
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: littleAdventure.x86_64
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --persist
```

```suggestion
  - --persist=.local/share/godot/app_userdata/LittleAdventure
```

Persist needs an argument.

This should allow the game settings to persist accross launches.